docs/BR.md

```diff
@@ -2089,10 +2091,10 @@ Table: Policy Restricted
 
 Table: Permitted `policyQualifiers`
 
-| __Qualifier ID__                     | __Presence__ | __Field Type__ |  __Contents__ |
+| __policyQualifier__                     | __Presence__ | __Field Type__ |  __Contents__ |
```

Comment:
This should likely be `policyQualifierId` as we're talking about `policyQualifierId` in `PolicyQualifierInfo` within `policyQualifiers`.

Reply:
I agree with this change. Thanks.